core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/ast/PropertyShape.java

```diff
@@ -187,12 +187,12 @@ public PlanNode generateTransactionalValidationPlan(ConnectionsGroup connections
 //
 //				PlanNode allTargetsPlan = getAllLocalTargetsPlan(connectionsGroup, negatePlan);
 //
-//				Unique invalid = new Unique(planNode);
+//				Unique invalid = Unique.getInstance(planNode);
```

Comment:
Does this commented out code block need to remain?

Reply:
It's the code to support pushing negation. I'd like to just leave it there for when/if I get back to reimplementing negation pushing.